### ISSUE TITLE
Modify the challenge configuration after the competition starts

### ISSUE THREAD

**Issue by Khinchine** [Cloud-CV/EvalAI#3976]
Can I modify the challenge configuration after the competition starts on eval.ai?  Currently, the results of Test-Standard2023 will not be displayed on the leaderboard. 
Can I make Test-Standard2023 public on the leaderboard after the competition starts?

**Comment by RishabhJain2018** [comment]
Yes, you can. Just make changes in your github configuration and it will be updated.
